<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="q_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>42</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_1">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>246</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_2">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>246</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 700 10pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_3">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>246</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 700 10pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_4">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>281</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_5">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>281</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_6">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>281</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_7">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>316</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_8">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>316</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_9">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>316</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_0">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>351</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_pushButton_1">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>211</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
font: 700 10pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_pushButton_2">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>246</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_pushButton_3">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>281</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_pushButton_4">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>316</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
font: 700 10pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result_pushButton">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>351</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_pushButton">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>211</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_pushButton">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>176</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="a_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>89</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;맑은 고딕&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="per_pushButton">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>351</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dot_pushButton">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>351</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;맑은 고딕&quot;;
background-color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gualhoL_pushButton">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>211</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
font: 700 10pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gualhoR_pushButton">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>211</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);
font: 700 10pt &quot;맑은 고딕&quot;;</string>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
